AdyenNetworking/APIClient/HTTPResponse.swift

```diff
@@ -17,6 +17,8 @@ public struct HTTPResponse<R: Response> {
     /// Response body
     public let responseBody: R
     
+    /// Response body captured as a binary representation
+    public let responseData: Data
```

Comment:
Could be optional?

Reply:
it could indeed, but i actually dont want to make this optional... if there is no response body, then an empty Data object will be returned to the caller

Reply:
Fair enough